### ISSUE TITLE
[bug]: blotchy images after converting ckpt/safetensors to diffusers

### ISSUE THREAD

**Issue by jonathanljs** [invoke-ai/InvokeAI#2671]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8GB

### What happened?

All diffuser models come out blotchy to various levels (original or converted) - see attached screenshot. Doesn't happen with Safetensors / CKPT models.

### Screenshots

![Screenshot 2023-02-15 114248](https://user-images.githubusercontent.com/55745688/219018496-3417dec3-258a-406d-9bee-84c293387cc8.jpg)

### Additional context

I'm on an older GPU (1070) and there are some models it can't cope with at all. Could this be the source of the problem?

### Contact Details

discordname jonathanljs#0290

**Comment by cwallen** [comment]
I've seen this as well though not nearly as bad as your screenshot. I think it only happens on some samplers.

**Comment by lstein** [comment]
Could you walk through the exact sequence of steps you took that leads to this result? So far I haven’t been able to replicate it?

**Comment by jonathanljs** [comment]
It was quite simple: I converted some ckpt/safetensor models to diffusers using the command line, and when I created a new image, that's what I got when doing txt2img or img2img. Also with the inbuilt diffuser models. On 2.3.0. Now I'm on 2.3.1 and diffusers still don't work, I just get semi-blank fog on txt2img and foggy versions of the original on img2img.
Some ckpt/safetensors also don't work on my system, often seems to be the fp32 ones. I'm thinking it might be because I'm using a GTX1070 and it just can't handle it..? (I have xformers running.)
don't know if it's connected, but Python 3.10 doesn't work in any form either, only 3.9. This is a Windows 11 machine.
